db/migrate/20210701000400_add_user_id_to_participant_declarations.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class AddUserIdToParticipantDeclarations < ActiveRecord::Migration[6.1]
+  disable_ddl_transaction!
```

Comment:
Why do we need this?

Reply:
It's from strong_migrations and required when doing the `index: { algorithm: concurrently }`